ichnaea/data/tests/test_export.py

```diff
@@ -213,12 +228,14 @@ def test_upload(self, celery, session, metricsmock):
             url="s3://bucket/backups/{source}/{api_key}/{year}/{month}/{day}",
         )
         ApiKeyFactory(valid_key="e5444-794")
+        ApiKeyFactory(valid_key="no-position")
         session.flush()
 
         reports = self.add_reports(celery, 3)
         self.add_reports(celery, 3, api_key="e5444-794", source="gnss")
         self.add_reports(celery, 3, api_key="e5444-794", source="fused")
         self.add_reports(celery, 3, api_key=None)
+        self.add_reports(celery, 3, api_key="no-position", set_position=False)
```

Comment:
This test confirms the `s3` exporter does not have an issue with `position: None`.